apps/meteor/client/sidebar/footer/voip/VoipFooter.tsx

```diff
@@ -132,6 +135,7 @@ export const VoipFooter = ({
 								small
 								square
 								danger
+								data-qa-id='omncVoipRejectButton'
```

Comment:
even data-qa been more permissive then `id` itself makes no sense to me have multiple data-qa-id with same value